src/reporter/tests/_timescale_tmp.py

```diff
@@ -57,7 +57,7 @@ def gen_notification(entities):
 
 
 def post_notification(payload, fw_svc=None, fw_path=None):
-    url = f"{QL_URL}/notify"
+    url = f"{QL_URL}/v2/notify"
```

Comment:
@daminichopra `QL_URL` in this file is defined at the top of this module:

    QL_URL = f"http://{QL_HOST}:{QL_PORT}/v2"

it already has the dreaded `v2` in it, so I guess we should undo this change?